tasks/update-code/git.yml

```diff
@@ -1,6 +1,30 @@
 ---
+
+- name: ANSISTRANO | GIT | Ensure GIT deployment key is up to date
+  copy:
+    src: "{{ ansistrano_git_identity_key_path }}"
+    dest: ".ssh/git_identity_key"
```

Comment:
I cannot see any problem with the change, but a clear advantage: if you have to open a terminal at the server and do some dirty stuff manually, you can be sure that the key file is the right one for that project. If @emmetog agree, I will open a PR.